### PR TITLE
enable downstream release handling via packit

### PR DESCRIPTION
**Pull request by sopos**
enables automated Fedora releases
- create downstream PRs for the release
- automatically create downstream Koji builds base on the merged PRs
- automatically created Bodhi updates for the builds

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -1,47 +1,61 @@
 specfile_path: dist/beakerlib.spec
-#synced_files:
-#    - beakerlib.spec
+
+files_to_sync:
+  - src: dist/*
+    dest: .
+  - .packit.yaml
```
Comment (psss) [inline .packit.yaml:6]: I guess you also want to sync specfile?

File: .packit.yaml [hunk 2]
```diff
@@ -1,47 +1,61 @@
...
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - fedora-rawhide
+    - fedora-branched
+    - epel-all
+
+- job: koji_build
+  trigger: commit
+  allowed_committers: ["sopos"]
```
Comment (psss) [inline .packit.yaml:51]: In `tmt` we also have `packit` included. Not sure if it's needed, @lachmanfrantisek will know for sure.

**Review by psss — COMMENTED** [review]
Looks good, just two comments.

File: .packit.yaml (hunk 1 above)
Comment (sopos) [reply .packit.yaml:6]: the spec file is in the dist directory, so it should be covered

Comment (psss) [reply .packit.yaml:6]: So, you're not building (or planning to build) `rpms` for pull requests?

File: .packit.yaml (hunk 2 above)
Comment (lachmanfrantisek) [reply .packit.yaml:51]: The `packit` is in[ the `allowed_pr_authors`.](https://packit.dev/docs/configuration/downstream/koji_build#optional-parameters), So no need to have it there.

File: .packit.yaml [hunk 3]
```diff
@@ -1,47 +1,61 @@
...
-    targets:
+  targets:
+  - fedora-all
+  - epel-8
+
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - fedora-rawhide
+    - fedora-branched
```
Comment (lachmanfrantisek) [inline .packit.yaml:46]:
```suggestion
    - fedora-all
```

This should be the same.

File: .packit.yaml (hunk 2 above)
Comment (sopos) [reply .packit.yaml:51]: I see I just copied the example and tweaked it. I will rather replace it with the `allowed_pr_authors`, it make much more sense in this workflow.

File: .packit.yaml (hunk 1 above)
Comment (sopos) [reply .packit.yaml:6]: I do not think it is necessary to build on downstream PR while we build on the upstream PR already. Respectively, the build will be triggered based on the merge of the PR.

Comment (sopos) [reply .packit.yaml:6]: I mean, the spec file is present in the upstream `/dist` directory so it gets synced to the downstream from there. And while have the same spec, I consider the upstream CI sufficient.

Comment (psss) [reply .packit.yaml:6]: Ah, I see! I somehow haven't noticed the `dist` directory. I thought you were speaking about the dist git repo. Now everything makes sense ;)

File: .packit.yaml [hunk 4]
```diff
@@ -1,47 +1,60 @@
...
+      - epel-all-x86_64
+      - epel-all-aarch64
+
+  - job: tests
+    trigger: pull_request
     targets:
-    - fedora-all
-    - epel-8
+      - fedora-all
+      - epel-8
```
Comment (sopos) [inline .packit.yaml:40]:
```suggestion
      - epel-9
```

**Pull request merged**